skiko/src/jvmMain/objectiveC/macos/Drawlayer.mm

```diff
@@ -141,16 +141,6 @@ JNIEXPORT void JNICALL Java_org_jetbrains_skiko_HardwareLayer_nativeDispose(JNIE
     }
 }
 
-JNIEXPORT jlong JNICALL Java_org_jetbrains_skiko_HardwareLayer_getWindowHandle(JNIEnv *env, jobject canvas)
-{
-    LayerHandler *layer = findByObject(env, canvas);
-    if (layer != NULL)
-    {
-        return (jlong)[layer window];
```

Comment:
Let's remove layer.window and layerSet.window initialization inside nativeInit to avoid code duplication.

P.S.
I checked that performance of getting windowHandle every time isn't so bad:
```
repeat(10) {
    JFrame().apply {
        size = Dimension(200, 300)
        isVisible = true
    }
}
window.pack()
println(measureTime {
    repeat(100) {
        window.layer.windowHandle
    }
})
```
takes 6ms on MacBook Pro 2019 2.3 GHz i9

Reply:
Not sure is 6ms is good enough for such a basic op, but anyway.

Reply:
6ms take 100 calls of windowHandle

Reply:
> Let's remove layer.window and layerSet.window initialization inside nativeInit to avoid code duplication.
I think that finding NSWindow should only be implemented in one way